xcom2-launcher/xcom2-launcher/Forms/MainForm.Events.cs

```diff
@@ -141,6 +141,17 @@ internal void RegisterEvents()
             export_load_button.Click += ExportLoadButtonClick;
         }
 
+        private void QuickArgumentItemClick(object sender, EventArgs e) {
+            // Add or remove the respective argument from the argument list, depending on its check-state.
+            if (sender is ToolStripMenuItem item)
+            {
+                if (item.Checked)
+                    Settings.AddArgument(item.Text);
+                else
+                    Settings.RemoveArgument(item.Text);
+            }
```

Comment:
Maybe it makes sense to `else { throw }` or something? Dunno how it's usually done in winforms

Reply:
Probably a matter of taste in this case. But can not hurt, so someone who uses the handler in a wrong way gets "notified". Would actually be a good use case for an "Assertion". I will change it.